src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -13,23 +36,45 @@ const CreateBenchmarkPage = () => {
 
   const addSubgoalMutation = trpc.iep.addSubgoal.useMutation();
 
+  const VIEW_STATES = {
+    BENCHMARK_PG_1: 0,
+    BENCHMARK_PG_2: 1,
+  };
+
+  const [viewState, setViewState] = useState(VIEW_STATES.BENCHMARK_PG_1);
+
+  const [benchmarkFormState, setBenchmarkFormState] =
+    useState<BenchmarkFormState>({
+      description: undefined,
+      setup: undefined,
+      materials: undefined,
+      instructions: undefined,
+      baseline_level: undefined,
+      target_level: undefined,
+      attempts_per_trial: undefined,
+      number_of_trials: undefined,
+    });
+
+  const steps = ["Instructional Guidelines", "Data Collection Guidelines"];
+
```

Comment:
Great suggestion. I used the functions you wrote here. Instead of useEffect to disable the button, I opted for checks on click of the "Next" button and the submit button, so the user cannot move to the next page without filling out the fields in page one.